homeassistant/config_entries.py

```diff
@@ -30,7 +30,6 @@
 import voluptuous as vol
 
 from . import data_entry_flow, loader
-from .components import persistent_notification
```

Comment:
Removing this line makes the test `tests/test_circular_imports.py::test_circular_imports` fail